# Array of detected objects from YOLO
std_msgs/Header header

# Array of all detected objects in this frame
DetectedObject[] objects

